src/editors/data/services/cms/utils.ts

```diff
@@ -16,6 +16,15 @@ export const get: Axios['get'] = (...args) => client().get(...args);
  * @param {object|string} data - post payload
  */
 export const post: Axios['post'] = (...args) => client().post(...args);
+
+/**
+ * post(url, data)
+ * simple wrapper providing an authenticated Http client post action
+ * @param {string} url - target url
+ * @param {object|string} data - post payload
```

Comment:
```suggestion
```
These JSDoc are being ignored. Better just to not include them.

![Screenshot](https://github.com/user-attachments/assets/c669abb0-8c22-45d5-91a4-d4a0f5b74cb1)